packages/dts-critic/index.ts

```diff
@@ -233,34 +231,32 @@ Try adding -browser to the end of the name to get
  * Checks DefinitelyTyped npm package.
  * If all checks are successful, returns the npm version that matches the header.
  */
-function checkNpm(name: string, npmInfo: NpmInfo, header: headerParser.Header | undefined): NpmError | string {
+function checkNpm(name: string, npmInfo: NpmInfo, header: headerParser.Header): NpmError | string {
   if (!npmInfo.isNpm) {
     return {
       kind: ErrorKind.NoMatchingNpmPackage,
       message: `Declaration file must have a matching npm package.
 To resolve this error, either:
 1. Change the name to match an npm package.
-2. Add a Definitely Typed header with the first line
-
-
-// Type definitions for non-npm package ${name}-browser
-
-Add -browser to the end of your name to make sure it doesn't conflict with existing npm packages.`,
+2. Add \`"nonNpm": true\` to the package.json to indicate that this is not an npm package.
```

Comment:
Should we still suggest the `-browser` suffix as an example?

Reply:
Yeah, we can do this later. `-browser` leads to people tacking `-browser` to their name to silence the lint even if it doesn't make any sense... Saw that once already last week.

Reply:
Yeah, I felt that advice was very outdated at best